src/src/AdminBundle/Controller/MailingListController.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace AdminBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use AdminBundle\Entity\Attendee;
+use AdminBundle\Entity\Event;
+use AdminBundle\Entity\EventType;
+use AdminBundle\Entity\Registration;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Doctrine\ORM\Query\Expr\Join;
+
+class MailingListController extends Controller
+{
+    /**
+     * @Route("/mailing_list/{page}", name="mailing_list", defaults={"page"=1})
+     * @Method("GET")
+     */
+    public function indexAction($page, Request $request)
+    {
+        $numItemsPerPage = 3;
+        $countPages = 0;
+        $mailingList = [];
+        $em = $this->getDoctrine()->getManager();
+        $defaultData = array('nameEmail' => '', 'event' => '', 'isSubscribed' => false);
+        $form = $this->createFormBuilder($defaultData)
+            ->add('nameEmail', TextType::class, array(
+                'label' => 'Name / Email',
+                'required' => false,
+            ))
+            ->add('event', TextType::class, array(
+                'required' => false,
+            ))
+            ->add('isSubscribed', CheckboxType::class, array(
+                'label' => 'Only subscribed',
+                'required' => false,
+            ))
+            ->add('filter', SubmitType::class, array(
+                'label' => 'Filter',
+            ))
+            ->setAction($this->generateUrl('mailing_list'))
+            ->setMethod('GET')
+            ->getForm();
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid() && $this->filterIsNotEmpty($form)) {
+            $searchedNameEmail = $form->getData()['nameEmail'];
+            $searchedEvent = $form->getData()['event'];
+            $searchedIsSubscribed = $form->getData()['isSubscribed'];
+
+            if ($searchedEvent != "" ) {
+                $selectMailingList = $em->createQueryBuilder('r')
```

Comment:
querybuilder patri do repozitory. http://www.mendoweb.be/blog/using-repositories-doctrine-2/

Reply:
to plati pre vsetky v tomto PR

Reply:
Áno, voľakedy sme to nerobili ale ma sa to robiť už ani posledný PR od Alex som neapprovol kým to nespravila